src/privatesend.h

```diff
@@ -339,7 +339,6 @@ class CPrivateSendBaseSession
         finalMutableTransaction(),
         nSessionDenom(0)
         {}
-    CPrivateSendBaseSession(const CPrivateSendBaseSession& other) { /* dummy copy constructor*/ SetNull(); }
```

Comment:
Oh my :see_no_evil: 
Didn't even notice this one, good it's going away now :)

Reply:
I'm wondering this didn't cause problems due to the use of std::vector before. Shouldn't this have resetted all sessions from time to time?

Reply:
Yep, good point, but I think it was only the case on mixing start when we were pushing new sessions into the vector until vector had the desired size.